src/index.ts

```diff
@@ -72,10 +71,7 @@ export async function resolve(
   /**
    * If no extension is passed and is a relative import then let's try to find a `.ts` or `.tsx` file at the path
    */
-  if (
-    (relativePathRegex.test(specifier) || forceRelative) &&
-    !hasExtensionRegex.test(fileName)
-  ) {
+  if (relativePathRegex.test(specifier) && !hasExtensionRegex.test(fileName)) {
```

Comment:
I can remove my stupid `forceRelative` hack now that I've made it so it returns relative paths instead of absolute.